assets/src/apps/authoring/components/PropertyEditor/schemas/screen.ts

```diff
@@ -1,18 +1,142 @@
+import CustomFieldTemplate from '../custom/CustomFieldTemplate';
 const screenSchema = {
   type: 'object',
   properties: {
+    Size: {
+      type: "object",
+      title: "Dimensions",
+      properties: {
+        width: { type: 'number' },
+        height: { type: 'number' }
+      }
+    },
+    Position: {
+      type: "object",
+      title: "Position",
+      properties: {
+        x: { type: 'number' },
+        y: { type: 'number' },
+        z: { type: 'number' }
+      }
+    },
+    useHtmlProps: {
```

Comment:
this one is used to determine whether to use the "backgroundColor" type of settings in the palette or the legacy ones, we don't want to have this be an option in the authoring, we'll hard code it for new lessons. and possibly "convert" legacy ones that are opened in the editor

Reply:
we *do* still need to have those settings for "backgroundColor", "borderColor", "borderWidth", "borderRadius", "borderStyle" to exist